src/focus-strategies/focus-by-registry.ts

```diff
@@ -0,0 +1,41 @@
+import { Direction, IArcHandler } from '../model';
+export class FocusByRegistry {
```

Comment:
yeah definitely makes more sense. I will try to refactor those in a different PR